Helper/Api/Attributes.php

```diff
@@ -117,6 +125,9 @@ public function processProductAttributes(array $flat_product, Product $target, s
                 && array_key_exists('attribute_set_name', $flat_product)
             ) {
                 try {
+                    $attributeArray = $this->attributeApiClient->getAttributeById($storeId, $attribute['attribute_id']);
+                    $attribute['is_options'] = array_key_exists('is_options', $attributeArray) ? $attributeArray['is_options'] : null;
```

Comment:
```suggestion
if(is_null($attributeArray)) {
   continue;
}
                    $attribute['is_options'] = array_key_exists('is_options', $attributeArray) ? $attributeArray['is_options'] : null;
```

If we cannot get, maybe ignore?